VRDR/IJEMortality.cs

```diff
@@ -1077,21 +1077,46 @@ public string SEX_BYPASS
         [IJEField(15, 191, 9, "Social Security Number", "SSN", 1)]
         public string SSN
         {
+
             get
             {
+                string fhirFieldName = "SSN";
+                string ijeFieldName = "SSN";
+                IJEField info = FieldInfo(ijeFieldName);
```

Comment:
When we're dealing with only a single field (SSN in this case) we don't need to reference the property via FieldInfo, we can just access it directly (like you actually do with record.SSN) and use the numeric field lengths directly. We only need FieldInfo if we want to do this dynamically for an arbitrary string (like we do in the helper methods).